packages/dds/merge-tree/src/ordinal.ts

```diff
@@ -11,20 +11,21 @@ export function computeHierarchicalOrdinal(
 	actualCount: number,
 	parentOrdinal: string,
 	previousOrdinal: string | undefined,
-) {
+): string {
 	assert(
 		maxCount <= 16 && actualCount <= maxCount,
 		0x3f0 /* count must be less than max, and max must be 16 or less */,
 	);
 
 	const ordinalWidth = 1 << (maxCount - actualCount);
 	let ordinal: string;
-	if (previousOrdinal === undefined) {
-		ordinal = parentOrdinal + String.fromCharCode(ordinalWidth - 1);
+	if (previousOrdinal === undefined || previousOrdinal === "") {
```

Comment:
This appears to a behavioral change. Is this intentional / safe to make?

Reply:
This is intentional - to find the parent ordinal, the tests use fromCodePoint, which returns an empty string if the result of making the elements passed in into a string has a length of 0. fromCharCode returned undefined, but the tests were not accurately determining if there was a parentOrdinal without the extra check.